nemo/collections/tts/models/hifigan.py

```diff
@@ -58,32 +57,41 @@ def __init__(self, cfg: DictConfig, trainer: 'Trainer' = None):
         self.discriminator_loss = DiscriminatorLoss()
         self.generator_loss = GeneratorLoss()
 
+        self.l1_factor = 45
+        self.num_log_audios = 5
+
         self.sample_rate = self._cfg.preprocessor.sample_rate
         self.stft_bias = None
 
         if self._train_dl and isinstance(self._train_dl.dataset, MelAudioDataset):
-            self.finetune = True
-            logging.info("fine-tuning on pre-computed mels")
+            self.input_as_mel = True
         else:
-            self.finetune = False
-            logging.info("training on ground-truth mels")
+            self.input_as_mel = False
+
+        self.automatic_optimization = False
```

Comment:
What is this flag?

Reply:
It allows to use loss.backward() manually, as we need it in GAN. It shoudn't be in config, because it is always for HiFiGAN and moreover previous flag didn't work (no valid code).